endpoints/get.go

```diff
@@ -38,9 +46,16 @@ func NewGetHandler(storage backends.Backend, metrics *metrics.Metrics, allowCust
 
 func (e *GetHandler) handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	e.metrics.RecordGetTotal()
+
+	if utils.RandomPick(e.cfg.refererLogRate) == true {
+		if refererHeaderValue := r.Header.Get(utils.REFERER_HEADER_KEY); refererHeaderValue != "" {
+			log.Info("Incoming request Referer header: " + refererHeaderValue)
```

Comment:
Might be interesting to specify GET vs PUT in the log messages.

Reply:
Brilliant. Done